addons/mail/static/src/chatter/web_portal/chatter.js

```diff
@@ -274,11 +272,12 @@ export class Chatter extends Component {
      * @param {['activities'|'followers'|'attachments'|'messages'|'suggestedRecipients']} requestList
      */
     load(thread, requestList = ["followers", "attachments", "messages", "suggestedRecipients"]) {
-        if (!thread.id || !this.state.thread?.eq(thread)) {
+        if (!this.state.thread?.eq(thread)) {
             return;
         }
-        if (this.props.hasActivities && !requestList.includes("activities")) {
-            requestList.push("activities");
```

Comment:
I don't think removing this key should be part of this PR. We still want to specify depending on the flux whether we need to fetch activities or not.